src/content/docs/apm/agents/nodejs-agent/api-guides/nodejs-agent-api.mdx

```diff
@@ -859,16 +859,16 @@ New Relic's Node.js agent includes additional API calls.
   </Collapser>
 
   <Collapser
-    id="setUserID"
-    title={<InlineCode>newrelic.setUserID(string)</InlineCode>}
+    id="setUserId"
```

Comment:
this isn't correct. The API docs are (setUserID)[https://newrelic.github.io/node-newrelic/API.html#setUserID]

Reply:
This is what I see on the newrelic object in the browser after initializing the script:
<img width="630" alt="image" src="https://github.com/newrelic/docs-website/assets/42801244/119769b1-8f69-4d66-831f-1645b353b8a5">

I.e. with a small d. 
We are using  newrelic: "11.9.0" and @newrelic/next: "0.7.0".

Reply:
So maybe the browser function is spelled differently? A little cofusing.